16 Lazy Loading/src/index.tsx

```diff
@@ -2,19 +2,36 @@ import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import { Router, Route, IndexRoute, hashHistory } from 'react-router';
 import App from './components/app.tsx';
-import AboutPage from './components/about/aboutPage';
-import MembersPage from './components/members/membersPage';
 import MemberPage from './components/member/memberPage';
 
+const lazyLoadAboutComponent = () => {
+  return {
+      getComponent: (loc, cb)=> {
+        require.ensure([], require => {
+          cb(null, require('./components/about/aboutPage')["default"]);
+        }, 'AboutPage');
+      }
+    }
+};
+
+const lazyLoadMembersComponent = () => {
```

Comment:
Good stuff, just adding something on top to enhance the sample, could it be a good idea to try to group members and member page in a single chunk, buy doing this:  We will show how to do lazy loading of a simple page (About) and how to do lazy loading of a group of related pages (member + members), we can add a comment indicating this.